#!/bin/sh -e
## 02_utilspm.dpatch by Guido Trotter <ultrotter@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct utils.pm script to begin with #!/usr/bin/perl

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/ultrotter/docs/projects/deb/nagios/nagios-plugins-1.3.1/plugins-scripts/utils.pm.in nagios-plugins-1.3.1/plugins-scripts/utils.pm.in
--- /home/ultrotter/docs/projects/deb/nagios/nagios-plugins-1.3.1/plugins-scripts/utils.pm.in	2003-02-03 21:29:55.000000000 +0100
+++ nagios-plugins-1.3.1/plugins-scripts/utils.pm.in	2004-02-01 10:42:54.000000000 +0100
@@ -1,3 +1,4 @@
+#!/usr/bin/perl
 # Utility drawer for Nagios plugins.
 # $Id: 02_utilspm.dpatch 469 2005-09-30 13:15:15Z seanius $
 #
